Maths/IsEven.js

```diff
@@ -0,0 +1,19 @@
+/* Even Number: https://simple.wikipedia.org/wiki/Even_number
+ *
+ * function to check if number is even
+ * return true if number is even
+ * else false
+ */
+
+/**
+ * @param {number} number
+ * @return {boolean}
+ */
+
+const isEven = (number) => {
+  return (number & 1) === 0
```

Comment:
Add an explanation for this line, as many people are not familiar with bitwise operators. For example:
```js
/*
* Bitwise AND (&) compares the bits of the 32 bit
* binary representations of the number and
* returns a number after comparing each bit like so:
*
* 0 & 0 -> 0
* 0 & 1 -> 0
* 1 & 0 -> 0
* 1 & 1 -> 1
*
* For odd numbers, the last binary bit will be 1 and
* for even numbers, the last binary bit will be 0.
*
* As the number is compared with one, all the
* other bits except the last will become 0. The
* last bit will be 0 for even numbers and 1 for
* odd numbers.
*/
```